### PR TITLE
Fixes 2230

### PR DESCRIPTION
**Pull request by pmbohm**
Corresponding Chef changes wont be required by AODN but this is could be of interest to @markhepburn and or @fxmzb123  ?

### CODE REVIEW

File: grails-app/views/wps/jobComplete.gsp [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Hi,
 
-Your IMOS subset has been created. You can download it by clicking here: ${jobReportUrl}
+Your ${grailsApplication.config.portal.siteAcronym} subset has been created. You can download it by clicking here: ${jobReportUrl}
```
Comment (jonescc) [inline grails-app/views/wps/jobComplete.gsp]: @pmbohm - refer https://github.com/aodn/aodn-portal/pull/2220 - grailsApplication is not available in this template and the above results in a null pointer exception.

Comment (jonescc) [inline grails-app/views/wps/jobComplete.gsp]: Perhaps pass the config as a parameter

File: grails-app/services/au/org/emii/portal/WpsService.groovy [hunk 2]
```diff
@@ -49,15 +49,26 @@ class WpsService extends AsyncDownloadService {
...
-        mailService.sendMail {
-            async true
-            to params.email.to
-            subject params.email.subject
-            body(
-                view: "/wps/${params.email.template}",
-                model: params
+        params.portalConfig = grailsApplication.config.portal
+
+        if (grailsApplication.config.mail.disabled == true) {
```
Comment (jonescc) [inline grails-app/services/au/org/emii/portal/WpsService.groovy]: Shouldn't it be the other way around?

Comment (pmbohm) [inline grails-app/services/au/org/emii/portal/WpsService.groovy]: `grailsApplication.config.mail.disabled` never existed either, so i always got what i expected!

**Comment by pmbohm** [comment]
Fixes #2230

File: grails-app/services/au/org/emii/portal/WpsService.groovy [hunk 3]
```diff
@@ -49,15 +49,26 @@ class WpsService extends AsyncDownloadService {
...
+                subject params.email.subject
+                body(
+                    view: "/wps/${params.email.template}",
+                    model: params
+                )
+            }
+        }
+        else {
+            // email is disabled for dev's
+            println (
```
Comment (jonescc) [inline grails-app/services/au/org/emii/portal/WpsService.groovy]: Perhaps use a logging statement here instead so output can be controlled by logging framework like other logging statements.

**Comment by jonescc** [comment]
Looks good in vagrant - just one comment re using a println.

**Comment by pmbohm** [comment]
`log.info` works fine. It always has and always will, but I'll need to be aware maybe it wont work in all files. @jonescc

**Pull request merged**